### DISCUSSION TITLE
Client Connection Pooling

### DISCUSSION THREAD

**Discussion by ghost** [appy-one/acebase-client#23] [General]
I am using AceBase with Node-Red. Does the client do connection pooling if I create multiple AceBaseClient objects in the same process? Trying to figure out if I need to create and share one client instance, or whether I can create one in each flow where I need it.

**Comment by appy-one** [comment]
No connection pooling is currently used, each instance has its own connection to the server

**Reply by ghost** [reply]
Thank you for the confirmation.
